README.md

```diff
@@ -1,9 +1,115 @@
+<div align="center">
+  <a href="https://inception-project.github.io/">
+    <img width="471" style="max-width:50%;" src="inception-doc/src/main/resources/META-INF/asciidoc/inception-logo.png"  alt="INCEpTION Logo"/>
+  </a>
+  <p>
+    A semantic annotation platform offering intelligent assistance and knowledge management.
+  </p>
+  <br/>
+  <p>
+    <a href="https://inception-project.github.io/"><strong>Homepage</strong></a> ·
+    <a href="https://inception-project.github.io/documentation/latest/user-guide"><strong>Usage</strong></a> ·
+    <a href="https://morbo.ukp.informatik.tu-darmstadt.de/demo"><strong>Demo</strong></a>  ·  
+    <a href="https://inception-project.github.io/documentation/latest/user-guide#sect_faq"><strong>FAQ</strong></a>
+  </p>
+  <p>
+
 [![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/inception-project/inception)](https://github.com/inception-project/inception/releases/latest)
 [![GitHub license](https://img.shields.io/github/license/inception-project/inception)](https://github.com/inception-project/inception/blob/master/LICENSE.txt)
-[![Join the chat at https://gitter.im/inception-project/Lobby](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/inception-project/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
+[![Join the chat at https://gitter.im/inception-project/Lobby](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/inception-project/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+  </p>
+
+</div>
+
+---
 
 # INCEpTION
 
 INCEpTION provides a semantic annotation platform offering intelligent annotation assistance and knowledge management.
+For more information, visit the [INCEpTION website](https://inception-project.github.io/). For a first impression on
+what INCEpTION is, you may want to watch
+our [introduction videos](https://www.youtube.com/watch?v=Ely8eBKqiSI&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-).
+
+<div align="center" style="margin: 10px">
+    <img src="https://inception-project.github.io/images/screenshot-annotation.png" alt="INCEpTION Screenshot" />
+</div>
+
+INCEpTION is a text-annotation environment useful for various kinds of annotation tasks on written text. Annotations are
+usually used for linguistic and/or machine learning concerns. INCEpTION is a web application in which several users can
+work on the same annotation project, and it can contain several annotation projects at a time. It provides a *
```

Comment:
formatting of *recommender system* is off; should also explain the term recommender because it is a technical for Inception specifically.